samples/python/_doc.py

```diff
@@ -17,11 +15,11 @@
     for fn in glob('*.py'):
         loc = {}
         try:
-            if PY3:
```

Comment:
Why this check doesn't work?

Reply:
The linters are set up to track the Python porting best practice [use feature detection instead of version detection](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection) so they look for try / except blocks with the appropriate exception (NameError in this case).